### DISCUSSION TITLE
RP2040 USB host connection to USB memory stick

### DISCUSSION THREAD

**Discussion by jeroentaverne** [hathach/tinyusb#1988] [Q&A]
Did anyone get this working? I am getting a panic over the UART when I connect any USB memory stick using the host_cdc_msc_hid example. Connecting an USB keyboard works fine. I will try to get some extra log output if this is not a known problem.

**Comment by hathach** [comment]
what is your tinysb log with LOG=2 (CFG_TUSB_DEBUG=2), maybe there is an incompatible issue with usb stick, try another one. The example `msc_file_explorer` should also navigate and do some basic filesystem modification as well last time I tested.

**Reply by jeroentaverne-dv** [reply]
I got it working. This issue can be closed, thank you. :-)
